app/app.go

```diff
@@ -59,8 +59,8 @@ import (
 )
 
 const (
-	AccountAddressPrefix = "cosmos"
-	Name                 = "poktroll"
+	AccountAddressPrefix = "pokt"
```

Comment:
I've noticed addresses no longer start with `pokt`, but with `cosmos` instead. Unless this is the desired behavior, I'm changing them back.